### PR TITLE
Dockerized bazel image. 

### PR DESCRIPTION
**Pull request by codecakes**
This is a WIP feature. The goal is to fully automated and integrate with Deployment over google cloud builder.

- Using custom python-buster image for bazel docker io rules.
- Updated Readme to reflect the changes made.
- Drops support for python v<3.8.x

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/covidx/36)

### CODE REVIEW

File: covidX/settings.py [hunk 1]
```diff
@@ -96,11 +96,13 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-if os.getenv("GAE_APPLICATION", None):
+if GAE_APPLICATION := os.getenv("GAE_APPLICATION", False) and (
```
Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (F999)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626&bid=20463343)

File: covidX/settings.py (hunk 1 above)
Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (<string>, line 99)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626&bid=20463343)

File: covidX/settings.py (hunk 1 above)
Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (F999)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (<string>, line 99)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [os.getenv default type is builtins.bool. Expected str or None.](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

File: covidX/settings.py [hunk 2]
```diff
@@ -96,11 +96,13 @@
...
+    CONNECTION_NAME := os.getenv("CONNECTION_NAME", False)
```
Comment (codecakes) [inline covidX/settings.py:100]: Codacy found an issue: [os.getenv default type is builtins.bool. Expected str or None.](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

**Comment by codecakes** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626&bid=20463343)

File: covidX/settings.py [hunk 3]
```diff
@@ -96,11 +96,13 @@
 # Database
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
-if os.getenv("GAE_APPLICATION", None):
+if GAE_APPLICATION := os.getenv("GAE_APPLICATION", None) and (
```
Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (F999)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

Comment (codecakes) [inline covidX/settings.py:99]: Codacy found an issue: [invalid syntax (<string>, line 99)](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6289626)

**Review by atuljain — APPROVED** [review]
Seems fine but take care of linting

**Pull request merged**